<launch>
  
  <!-- joynode -->
  <node pkg="joy" 
        exec="joy_node"
        name="joy_node"
        output="screen">
    <!-- <param name="dev" value="/dev/input/js0"/> -->
    <param name="deadzone" value="0.05"/>
  </node>


  <arg name="advanced_teleop_param"  default="$(find-pkg-share joy_manager)/config/advanced_joy.param.yaml"/>
  <!-- joy manager node-->
  <node pkg="joy_manager"
        exec="advanced_joy_node"
        name="advanced_joy_node"
        output="screen">
        
    <param from="$(var advanced_teleop_param)"/>
    <!-- joy -->
    <remap from="/joy" to="/joy"/>
    <!-- 自律走行　コマンド -->
    <remap from="/ackermann_cmd" to="/ackermann_cmd"/>
    <!-- jetracerへの入力 -->
    <remap from="/cmd_drive" to="/jetracer/cmd_drive"/> 
    <!-- rosbag制御トリガ -->
    <remap from="/rosbag2_recorder/trigger" to="/rosbag2_recorder/trigger"/>
    
  </node>

</launch>
